pkg/api/v1/types.go

```diff
@@ -646,14 +646,14 @@ type VirtualMachineInstancePresetSpec struct {
 	// Required.
 	Selector metav1.LabelSelector `json:"selector"`
 	// Domain is the same object type as contained in VirtualMachineInstanceSpec
-	Domain *DomainPresetSpec `json:"domain,omitempty"`
+	Domain *DomainSpec `json:"domain,omitempty"`
```

Comment:
it's weird that the preset spec didn't originally use the DomainSpec directly, especially since they contain all the same values.

@stu-gott Was there a reason for the separate DomainPresetSpec?